integration-test-groups/azure/azure-key-vault/src/test/java/org/apache/camel/quarkus/component/azure/key/vault/it/AzureKeyVaultWithIdentityTestProfile.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.azure.key.vault.it;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+public class AzureKeyVaultWithIdentityTestProfile implements QuarkusTestProfile {
```

Comment:
Is this used anywhere?

Reply:
Hi, yes. In the AzureKeyVaultWithIdentityTest see https://github.com/apache/camel-quarkus/pull/6985/files#diff-fd652ef1cd5698dc5c802b6db2cef36ee0747ca579cd4a4b2b275b3f593837bdR33